var/spack/repos/builtin/packages/hdf5-vol-async/package.py

```diff
@@ -7,7 +7,9 @@
 
 
 class Hdf5VolAsync(CMakePackage):
-    """This package enables asynchronous IO in HDF5."""
+    """This package enables asynchronous IO in HDF5.  WARNING: Please refer to the
+    documentation.  This package will be available in HDF5_PLUGIN_PATH, but in order to
+    use it the consumer must set the HDF5_VOL_CONNECTOR environment variable."""
```

Comment:
Please simplify the comment block:

"USAGE: Set the following environment variable [1]." 

  `$export HDF5_VOL_CONNECTOR="async under_vol=0;under_info={}"`

[1] https://hdf5-vol-async.readthedocs.io/en/latest/gettingstarted.html#set-environmental-variables
"